ElectronClient/InteropServiceHelper.js

```diff
@@ -92,6 +95,16 @@ class InteropServiceHelper {
 		return this.exportNoteTo_('printer', noteId, options);
 	}
 
+	static async defaultFilename(noteIds, fileExtension) {
+		const note = await Note.load(noteIds[0]);
+		const folder = await Folder.load(note.parent_id);
+
+		// friendlySafeFilename assumes that the file extension is added after
+		const filename = friendlySafeFilename(`${folder.title}-${note.title}`);
```

Comment:
Maybe `friendlySafeFilename(folder.title, 100) + ' - ' + friendlySafeFilename(note.title, 100)` in case the folder title is particularly long?